### PR TITLE
Add option to specify cusotm part file name

### PR DESCRIPTION
**Pull request by alvinlin123**
It would be useful to allow customization on the "-blessed" part of the file name to something else so that we can potentially have different part name (using UUID for example) for different build.

### CODE REVIEW

**Comment by adam-lynch** [comment]
Thanks for this. I guess the option should be called `suffix`. 

I could see someone asking for more later though. Let's say they don't like how it uses a hyphen or the way the number is put on the end. What if this option could accept a function which would be called to generate the suffix for each file? It could be used like this:

```javascript
.pipe(bless({
  suffix: function(index){
    return index + '_customSuffix';
  }
})
```

What do you think? This could be overthinking it.

**Comment by alvinlin123** [comment]
I don't see why not. I think it's a good idea to accept a function. I will change the config to be called "suffix" and can be either a string or function. Thanks for the great suggestion.

File: index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+var _               = require("lodash");
```
Comment (adam-lynch) [inline index.js]: You aren't using a lot of lodash so to save the end user having to download it all on install, we could depend on the individual methods instead (which are independently published on npm). If you want `_.isString` for example, we could depend on [lodash.isstring](https://www.npmjs.com/package/lodash.isstring) and `require('lodash.isstring')` here.

Comment (alvinlin123) [inline index.js]: ah, thanks for pointing that out, I did not know you can do that! That's awesome. I will make the change.

Comment (alvinlin123) [inline index.js]: Done :) let me know if anything else.

**Comment by adam-lynch** [comment]
Thanks a lot @alvinlin123! 👏 3.2.0 is now published with this feature.

**Pull request merged**